packages/account/src/Sections/Security/LoginHistory/list-cell.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+
+type TListCell = {
+    title: JSX.Element | string;
+    text: JSX.Element | string;
+    className?: string;
+    align?: 'left' | 'right';
+};
+
+const ListCell = ({ title, text, className, align }: TListCell) => (
+    <React.Fragment>
+        <Text as='h3' align={align} weight='bold' className='login-history__list__row__cell--title'>
+            {title}
+        </Text>
+        <Text
+            className={classNames(className, { 'login-history__list__row__cell--right': align === 'right' })}
```

Comment:
Lets remove this extre space { 'login-history__list__row__cell..

Reply:
This space is auto adjusted by prettier 😅